Reemit.Decompiler.Clr/Disassembler/StreamDisassembler.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Reemit.Common;
+using System.Collections.Immutable;
+
+namespace Reemit.Decompiler.Clr.Disassembler;
+
+public class StreamDisassembler(Stream stream)
```

Comment:
Going to leave this one as is for now because the plan is to ultimately have `StreamDisassembler` take metadata tables via ctor for use in disassembling tokens. Not sure what exactly that will look like just yet, but based on my reading the spec, I believe we'll be able to construct the `StreamDisassembler` after metadata tables are parsed, then pass it around to disassemble any method body.